pages/linux/compgen.md

```diff
@@ -0,0 +1,23 @@
+# compgen
+
+> A built-in command for auto-completion in bash, which is called on pressing TAB key twice.
+
+- List all commands that you could run:
```

Comment:
All the commands that you could run? Perhaps something like `List all commands for which completions are registered` or something might work better?

Reply:
@sbrl I am a bit doubtful on whether __completion__ is too technical a word for end user though? This command in essence prints all commands that the terminal accepts.

Reply:
Ah, ok. Let's leave it as-is then :-)